Shaders/StaticModel.frag

```diff
@@ -2,11 +2,13 @@
 
 out vec4 outColour;
 
-in vec2 passTexCoords;
+in vec3 passTexCoords;
 in vec3 passColour; 
 
+uniform sampler2DArray textureArray;
+
 void main()
 {
-	outColour = vec4(passColour, 1.0);
+	outColour = texture(textureArray, vec3(passTexCoords.x, passTexCoords.y, passTexCoords.z)) * vec4(passColour, 1.0);
```

Comment:
this can be 
`outColour = texture(textureArray, passTexCoords) * vec4(passColour, 1.0);`